.github/workflows/qa.yaml

```diff
@@ -148,14 +148,15 @@ jobs:
       - name: Run tests (with race detector)
         run: |
           go test -race ./...
+
```

Comment:
sneaky :p (but yeah, was missing)

Reply:
ghghg, well... Another commit was the right place, but forgive me for have been lazy :stuck_out_tongue: